frontend/src/features/mapSideBar/components/MapSideBarLayout.tsx

```diff
@@ -44,6 +53,18 @@ const Title = styled.span`
   text-align: left;
 `;
 
+/** Shortens property name to a reasonable length as some are quite long */
+const formattedName = (name?: string) => {
+  if (name) {
```

Comment:
Can you just do `return name?.substring(0, 80)`?

Reply:
created an AbbreviatedText component last sprint for this purpose.

Reply:
oh shoot I'll use that thanks!